registration/include/pcl/registration/ndt.h

```diff
@@ -483,9 +496,16 @@ class NormalDistributionsTransform : public Registration<PointSource, PointTarge
    * distribution, Equation 6.8 [Magnusson 2009]. */
   double gauss_d1_, gauss_d2_;
 
-  /** \brief The probability score of the transform applied to the input cloud,
+  /** \brief The likelihood score of the transform applied to the input cloud,
    * Equation 6.9 and 6.10 [Magnusson 2009]. */
-  double trans_probability_;
+  union {
+    PCL_DEPRECATED(1,
+                   16,
+                   "The member `trans_probability_` is deprecated. Please use "
+                   "`trans_likelihood_` instead.")
+    double trans_probability_;
```

Comment:
PCL_DEPRECATED add some of the message in the macro. Maybe reword it slightly?

Reply:
I agree, will push another commit when the other questions are resolved.